node/src/components/contract_runtime.rs

```diff
@@ -139,6 +139,7 @@ pub(crate) struct ContractRuntime {
 
     /// Finalized blocks waiting for their pre-state hash to start executing.
     exec_queue: ExecQueue,
+    system_contract_registry: SystemContractRegistry,
```

Comment:
Please add a comment as to why we need this member variable, since it's not obvious.